manifests/profile/server.pp

```diff
@@ -93,18 +85,27 @@
     'owner'  => 'root',
     'group'  => 'root',
     'mode'   => '0755',
+    'tag'    => 'st2::server',
+  })
+
+  ensure_resource('file', '/var/run/st2', {
```

Comment:
Don't like the idea of duplicating actions that were already done by the OS packages.

This also makes things harder to sync if something like permissions are changed in deb or rpm.